Makefile-test.mk

```
@@ -55,7 +55,17 @@ endif
 
 # Folder where the e2e tests are located.
 E2E_TARGET ?= ./test/e2e/...
-E2E_KIND_VERSION ?= kindest/node:v1.32.0
+E2E_K8S_VERSION ?= 1.32
+ifeq ($(E2E_K8S_VERSION),1.30)
+	E2E_K8S_VERSION_PATCH ?= 10
+else ifeq ($(E2E_K8S_VERSION),1.31)
+	E2E_K8S_VERSION_PATCH ?= 6
+else ifeq ($(E2E_K8S_VERSION),1.32)
+	E2E_K8S_VERSION_PATCH ?= 3
+else
+	E2E_K8S_VERSION_PATCH ?= 0
+endif
+E2E_KIND_VERSION ?= kindest/node:v$(E2E_K8S_VERSION).$(E2E_K8S_VERSION_PATCH)
```

Can we go on this way?

```suggestion
E2E_K8S_VERSION ?= 1.32.3
E2E_KIND_VERSION ?= kindest/node:v$(E2E_K8S_VERSION)
```